pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1430,6 +1449,34 @@ CREATE OR REPLACE VIEW SCHEMA_INFO.label AS
       AS values
   FROM SCHEMA_CATALOG.label_key lk;
 
+CREATE OR REPLACE VIEW SCHEMA_INFO.system_stats AS
+    SELECT
+    (
+        SELECT SCHEMA_PROM.approximate_row_count('_prom_catalog.series'::REGCLASS)
+    ) AS num_series_approx,
+    (
+        SELECT count(*) FROM SCHEMA_CATALOG.metric
+    ) AS num_metric,
+    (
+        SELECT count(*) FROM SCHEMA_CATALOG.label_key
+    ) AS num_label_keys,
+    (
+        SELECT count(*) FROM SCHEMA_CATALOG.label
+    ) AS num_labels;
+
+CREATE OR REPLACE VIEW SCHEMA_INFO.metric_stats AS
+    SELECT metric_name,
+    SCHEMA_PROM.approximate_row_count(format('prom_series.%I', table_name)::regclass) AS num_series,
+    (SELECT row_estimate FROM public.hypertable_approximate_row_count(format('prom_data.%I',table_name)::regclass)) AS num_samples_approx
+    FROM SCHEMA_CATALOG.metric ORDER BY metric_name;
+
+CREATE OR REPLACE VIEW SCHEMA_INFO.label_key_stats AS
+    SELECT k.key AS label_key,
+    COALESCE(cnt, 0) AS cardinality
```

Comment:
actually should this column simply be added to the existing `SCHEMA_INFO.label` view? Seems like this would belong there.

Reply:
sure, that would be better